Detections/SigninLogs/SigninBruteForce-AzurePortal.yaml

```diff
@@ -33,8 +33,9 @@ query: |
   | where AppDisplayName contains "Azure Portal"
   // Split out failure versus non-failure types
   | extend FailureOrSuccess = iff(ResultType in ("0", "50125", "50140"), "Success", "Failure")
-  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), makeset(IPAddress), makeset(OS), makeset(Browser), makeset(City), 
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), IPAddress = makeset(IPAddress), makeset(OS), makeset(Browser), makeset(City), 
   makeset(ResultType), FailureCount = countif(FailureOrSuccess=="Failure"), SuccessCount = countif(FailureOrSuccess=="Success") 
   by bin(TimeGenerated, authenticationWindow), UserDisplayName, UserPrincipalName, AppDisplayName
   | where FailureCount >= failureCountThreshold and SuccessCount >= successCountThreshold
-  | extend timestamp = StartTimeUtc, AccountCustomEntity = UserPrincipalName
+  | mvexpand IPAddress
+  | extend timestamp = StartTimeUtc, AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress 
```

Comment:
i don't think IPCustomEntity handles sets?

Reply:
we use mvexpand to bring this back to single IPAddress per line.  Tested on more than one workspace with varied data, we should be good here.